README.md

```diff
@@ -21,7 +21,9 @@ The [`asp-net-mvc` solution](demos/asp-net-mvc) demonstrates the use of ASP.NET
 
 ### Install the pre-requisites
 
-To complete the exercises, you require a Windows machine and Visual Studio. Please ensure you have followed these steps:
+To complete the exercises, you require a Windows machine and Visual Studio. You must be using a Windows client edition, such as Windows 10, rather than a server edition, such as Windows Server 2016. The Particular Platfom Installer does not work on server editions of Windows.
+
```

Comment:
Should we say is not supported on server editions instead doesn't not work? that in my mind sounds like a bug